app/services/the_tvdb_service/row.rb

```diff
@@ -0,0 +1,34 @@
+class TheTvdbService
+  class Row
+    attr_reader :media_id, :media_type, :data
+
+    def initialize(media_id, data)
+      @media_id = media_id
+      @data = data
+      # not sure if I am going to actually need this.
+      @media_type = 'Anime'
+    end
+
+    def update_episode!
+      episode = Episode.first_or_initialize(
+        media_id: media_id,
+        media_type: media_type,
+        number: data['absoluteNumber']
+      )
+
+      episode.season_number ||= data['airedSeason']
+      episode.relative_number ||= data['airedEpisodeNumber']
+      # was returning an empty string and not updating.
```

Comment:
You can use a `.blank?` check instead of `.nil?` to treat empty strings as missing